.gitlab/workflows.yml

```diff
@@ -46,28 +41,98 @@ Python Wheels:
   tags:
     - $runner
   interruptible: false
+  artifacts:
+    paths:
+      - artifacts
+    expire_in: 1 day
   script:
-    - echo "Nothing implemented yet."
+    - export PROJECT_SRCS_DIR=$PWD/.. 
+    # Install GitHub CLI
+    - dnf config-manager --add-repo https://cli.github.com/packages/rpm/gh-cli.repo
+    - dnf install -y gh jq
+    - upstream_url=https://github.com/NVIDIA/cuda-quantum && export GH_TOKEN=$GITHUB_TOKEN
+    - if [ -z "$(git remote -v | grep upstream)" ]; 
+      then git remote add upstream ${upstream_url}.git;
+      fi
+    # Retrieve the upstream CUDA Quantum artifacts
+    - digest=${CI_COMMIT_TITLE#ghcr.io/nvidia/$staged@}
+    - build_info=`cat deployments/staging/$staged/${digest#sha256:}`
+    - artifacts_url=`echo "$build_info" | grep -s -o 'artifacts-url:\s*\S*' | rev | cut -d ' ' -f 1 | rev`
+    - artifacts=$(gh api $artifacts_url -q '.artifacts[] | {name:.name, url:.archive_download_url}')
+    - mkdir -p /tmp/wheels
+    - |
+      for artifact in `echo "$artifacts"`; do 
+        name=`echo $artifact | jq -r '.name'`
+        if [ "${name#pycudaq-}" != "$name" ]; then
+          url=`echo $artifact | jq -r '.url'`
+          gh api $url > _pycudaq.zip
+          unzip -d _pycudaq _pycudaq.zip && cd _pycudaq
+          for wheel in `ls cuda_quantum-*-cp310-cp310-manylinux_*_$(uname -m).whl 2> /dev/null`; do
+            echo "Adding wheel $wheel."
+            mv $wheel "/tmp/wheels/$wheel"
+          done
+          cd .. && rm -rf _pycudaq*
+        fi
+      done
+    - export python=python3.10 
+    - cd /tmp/wheels/ && $python -m pip install cuda_quantum-*-cp310-cp310-manylinux_*_$(uname -m).whl
+    - cudaq_location=`$python -m pip show cuda-quantum | grep -e 'Location:\ .*$'`
+    - export CUDAQ_INSTALL_PREFIX="${cudaq_location#Location:\ }"
+    - cp -R $CUDAQ_INSTALL_PREFIX/lib64/cmake/fmt $CUDAQ_INSTALL_PREFIX/lib/cmake
+    - export EXATN_COMMIT=9c053ac1671c041827fee3aeb33cfc6bf43762eb
```

Comment:
Yea, they were there for testing/validation purposes only.
I've removed them in: https://github.com/NVIDIA/cuda-quantum/pull/643/commits/0fd94cf153ee254d5ff919d993054e5ceca7321b

Notes: these are not yet set as GitLab variables (pending review and merge of all the related PR's/MR's, etc.)

Here is the workflow run before those refs were remove: https://github.com/NVIDIA/cuda-quantum/actions/runs/6242053935 (docker build was failing since I bypassed the environment hence it tried to push to a wrong location)